crates/table/src/page.rs

```diff
@@ -671,6 +671,7 @@ impl<'page> VarView<'page> {
         // 2. The null granule is trivially initialized.
         // 3. The caller promised that `granule` is safe to overwrite.
         unsafe { self.write_chunk_to_granule(&hash.data, hash.data.len(), granule, PageOffset::VAR_LEN_NULL) };
+        var_len_obj.as_ref().len()
```

Comment:
Agree, there should be a separate metrics in future for blob storage usage. Indtroduced `blob_store_bytes` field is a table property, It should not be used to get info about `blob_stores` and should be independent of any optimisations made by `blob_store` implementations like refcounting.
I took your suggestion to document it better, I think that should do the job.

Reply:
I actually think that because we are measuring specifically the size of all of the data in the table, this implementation is correct. The actual physical memory usage should be a different and separate metric.